src/sentry/models/actor.py

```diff
@@ -1,10 +1,24 @@
+from collections import defaultdict, namedtuple
+from rest_framework import serializers
+
 from django.db import models
-from sentry.db.models import Model
 from django.db.models.signals import pre_save
 
+from sentry.db.models import Model
+from sentry.utils.compat import filter
+
+
 ACTOR_TYPES = {"team": 0, "user": 1}
 
 
+def actor_type_to_class(type):
+    # type will be 0 or 1 and we want to get Team or User
+    from sentry.models import Team, User
+
+    ACTOR_TYPE_TO_CLASS = {ACTOR_TYPES["team"]: Team, ACTOR_TYPES["user"]: User}
```

Comment:
This should still be declared at the module level so we're not building it every time.

Reply:
I had a problem with circular imports. Could probably refer to the model differently though

Reply:
Ahh ok, nbd then.